app/posts/smoking-cessation/2025/03/2025-03-20-smoking-cessation-pilot.md

```diff
@@ -0,0 +1,143 @@
+
+Currently, people who want to stop smoking tobacco can use:
+- in-person NHSE services via routes like their pharmacy or GP
+- digital apps
+- work out a plan on their own
+
+People who want more guided support but cannot access in-person services for any reason do not have an NHSE digital service to use.
+
+Availability of tobacco treatment services is not consistent across the country and does not reach populations with increasing tobacco dependency (younger members of the population, men, and those from more deprived backgrounds).
+
+A third of surveyed local authorities (33%) did not commission a universal service to help people stop smoking in 2022 (latest available data).
+
+Local Authority tobacco treatment services are mostly delivered face-to-face, which is harder for people who work shifts or travel for their work to access. These services are mostly accessed by the middle-to-older age groups, and fewer men accessing from younger age groups than women.
+
+Evidence from the national digital weight management service shows that digital services can engage more deprived populations and fill gaps in provision. This evidence comes from increased access and engagement rates across all indices of multiple deprivation (IMD) bands (measure of relative deprivation for small, fixed geographic areas of the UK).
+
+## Aim
+
+To create a digital pathway for tobacco cessation that helps people quit smoking while accessing the support they need, with a particular focus on those currently underserved by in-person services.
+
+## Our approach
+
+The service launch is split into two phases:
+
+Phase 1: Providing NHS Staff with access to digital support services via two external service providers, which will help us understand the efficacy of incentive-based models (models where providers are paid based on milestones reached by service users) and begin to evaluate the benefits of digital pathways.
+
+Phase 2: Providing NHS Staff with access to a dedicated Tobacco Dependency Hub (similar to the existing Weight Management Service) to continue evaluating the benefits of digital pathways, impact on targeted groups and desirability of a digital pathway (will introduce the third service provider).
+
+Our planned research activity would allow us to:
+
+1. Test the efficacy of the incentive-based payment model for providers and evaluate the quality of their services
+2. Better understand the needs to user cohorts to ensure they receive a service that meets their needs and expectations
+3. Ensure NHSE capture the right data to determine outcomes are being met
+
+## What we learned
+
+Our research and design activities focused on leveraging existing data and conducting interviews with stakeholders to ensure we have a strong foundational knowledge of the existing service, before conducting primary research.
+
+Our understanding of the Digital Tobacco Treatment (DTT) service was informed by conversations with:
```

Comment:
This DTT gets introduced quite suddenly - how does it relate to what's been written about so far?